packages/website/package.json

```diff
@@ -17,7 +17,7 @@
     "next-mdx-remote": "^4.4.1",
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
-    "waku": "workspace:*"
+    "waku": "0.19.0"
```

Comment:
I'm not sure how pnpm will deal with this in monorepo, you might set `npm:0.19.0` would be better?

```
By default, pnpm will link packages from the workspace if the available packages match the declared ranges. For instance, foo@1.0.0 is linked into bar if bar has "foo": "^1.0.0" in its dependencies and foo@1.0.0 is in the workspace.
```

https://pnpm.io/workspaces#workspace-protocol-workspace